sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -89,8 +89,6 @@
     public SentryClient(Connection connection, ContextManager contextManager) {
         this.connection = connection;
         this.contextManager = contextManager;
-
-        Sentry.setStoredClient(this);
```

Comment:
No longer statically storing `SentryClient` just because it was created. We only store it if you use `Sentry.init`.